src/main/java/com/blackducksoftware/integration/fortify/service/FortifyAttributeDefinitionApi.java

```diff
@@ -40,13 +40,20 @@
 
     private final static String FIELDS_ATTRIBUTE = "id,name,category,type,options,required";
 
-    private final static OkHttpClient.Builder okBuilder = getHeader(PropertyConstants.getFortifyUserName(),
-            PropertyConstants.getFortifyPassword());
+    private static OkHttpClient.Builder okBuilder;
```

Comment:
Can you please remove static and add final. Similar comments for other fields as well

Reply:
Made the changes we discussed. Could you please review again and approve.